cmd/buildah/addcopy.go

```diff
@@ -20,23 +20,24 @@ import (
 
 type addCopyResults struct {
 	addHistory       bool
+	authfile         string
+	blobCache        string
+	certDir          string
+	checksum         string
 	chmod            string
 	chown            string
-	checksum         string
-	quiet            bool
-	ignoreFile       string
 	contextdir       string
-	from             string
-	blobCache        string
+	creds            string
 	decryptionKeys   []string
+	excludes         []string
+	from             string
+	ignoreFile       string
+	quiet            bool
 	removeSignatures bool
-	signaturePolicy  string
-	authfile         string
-	creds            string
-	tlsVerify        bool
-	certDir          string
 	retry            int
 	retryDelay       string
+	signaturePolicy  string
+	tlsVerify        bool
```

Comment:
This makes it it harder to find the specific commits where fields were added.  Please don't do it.